package.json

```diff
@@ -10,6 +10,7 @@
     "eslint": "eslint src --ext \".js,.jsx\"",
     "eslint:ci": "yarn run eslint -f junit -o test-reports/lint-results.xml",
     "eslint:fix": "eslint src --ext \".js, .jsx\" --fix",
+    "lint": "yarn run eslint:ci && yarn run format:verify",
```

Comment:
not sure if we need that `-o test-reports/lint-results.xml` in eslint:ci , i would say for this purpose it would be better to keep that result in pipeline itself (quicker access etc.) and not put it into file, wdyt?

Reply:
Originally, it was created for internal GitLab repo, maybe there was some usage. Here and now I don't see any usage of lint reporting. If the code does not pass checks, we have to fix it. We don't need to build dashboards with errors.
You are right. Let me check if it works as expected

Reply:
<img width="937" alt="image" src="https://github.com/splunk/addonfactory-ucc-base-ui/assets/142901247/43502a35-db48-4304-b9fb-d5d98d671ff0">
it works fine!